packages/cli/src/databases/migrations/mysqldb/1695128658538-AddWorkflowMetadata.ts

```diff
@@ -0,0 +1,13 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+
+export class AddWorkflowMetadata1695128658538 implements ReversibleMigration {
+	transaction = false as const;
+
+	async up({ schemaBuilder: { addColumns, column } }: MigrationContext) {
+		await addColumns('workflow_entity', [column('meta').json]);
+	}
+
+	async down({ schemaBuilder: { dropColumns } }: MigrationContext) {
+		await dropColumns('workflow_entity', ['meta']);
+	}
+}
```

Comment:
Since the migration is identical across all three DBs, we should move this into the `common` folder instead.
[This](https://github.com/n8n-io/n8n/commit/413e0bccb4d0144e5bdcf9221762e3f2b4428194) is a good example of how this migration should be.

Reply:
alright thanks

Reply:
Updated